test/deprecated.ts

```diff
@@ -64,6 +67,9 @@ export class MessageName extends pb_1.Message {
     set me_deprecated(value: string) {
         pb_1.Message.setOneofField(this, 3, this.#one_of_decls[0], value);
     }
+    get has_me_deprecated() {
```

Comment:
has getters should have `@deprecated` tag as well.

Reply:
Good catch.  I was not aware of the `@deprecated` comments unit now.  The problem should be now fixed.